### PR TITLE
Crypto withdrawal

### PR DESCRIPTION
**Pull request by lubega-deriv**
### Added Changes ###

- [x] Added e2e test for crypto withdrawal module
- [x] Added logic to access verification code from qa box email to fetch withdrawal page content
- [x] Updated global c_login command to run updated scenario in test-app
- [x] Added config files from cypress 13.6.0 update

### Clickup Card ###

https://app.clickup.com/t/20696747/WALL-2848

### CODE REVIEW

File: cypress/support/e2e.js [hunk 1]
```diff
@@ -92,6 +92,7 @@ if (Cypress.env("oAuthToken") == "") {
       "large"
     )
     }
+    cy.findByRole('button', { name: 'Continue' }).click()
```
Comment (lubega-deriv) [inline cypress/support/e2e.js:95]: Additional step in c_login added as test-app now has a popup for SmartCharts after logging in

**Review by lubega-deriv — COMMENTED** [review]
Config files added in this PR to update cypress from 13.5.0 to 13.6.0

File: cypress/support/e2e.js (hunk 1 above)
Comment (markodonnell-deriv) [reply cypress/support/e2e.js:95]: We should be able to make this conditional. See how I've coded the wallets checkbox click in c_login

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
As per comment in PR 18, the cy.Origin code block should be reusable as a command (if outside the spec file) or function (if within).

File: cypress/support/e2e.js (hunk 1 above)
Comment (lubega-deriv) [reply cypress/support/e2e.js:95]: Updated

**Review by markodonnell-deriv — APPROVED** [review]
As discussed. Happy to merge now and refactor cy.Origin afterwards. We have a bit of a backlog of PRs now.

**Pull request merged**